src/connectors/README.md

```diff
@@ -23,8 +23,12 @@ import React, { Component } from 'react'
 import { withHistoryActions } from '@shopgate-ps/pwa-extension-kit/connectors';
 
 class MyComponent extends Component {
+  const params = {
```

Comment:
This is more cryptic now. For code examples it's better to show how it's used with real data, not passed props.

Reply:
Right on, updated with data.